Tests/PostgresKitTests/PostgresKitTests.swift

```diff
@@ -40,6 +40,18 @@ class PostgresKitTests: XCTestCase {
             }
         }
     }
+
+    func testCreateEnumWithBuilder() throws {
+        let conn = try PostgresConnection.test(on: self.eventLoop).wait()
+        defer { try! conn.close().wait() }
+
+        try conn.create(enum: "meal", cases: "breakfast", "lunch", "dinner").run().wait()
+        try conn.raw("DROP TYPE meal;").run().wait()
+
+        try conn.create(enum: SQLIdentifier("meal"), cases: "breakfast", "lunch", "dinner").run().wait()
+        try conn.raw("DROP TYPE meal;").run().wait()
```

Comment:
An issue for drop type support would be nice, too 👍

Reply:
Indeed. Now that the rest of this PR seems kosher, do you mind if I just tackle `DROP` support here before merging? I'll re-request review after.

Reply:
Sure thing. Either way works for me. I know forking can be a pain. FWIW, once you do three non-trivial PRs you can get contributor status which lets you write to branches. That should make adding new PRs easier. :) (https://discordapp.com/channels/431917998102675485/437050958061764608/443504296504590338)